### PR TITLE
Fix for no score available continuously popping toaster once score becomes available

### PR DESCRIPTION
**Pull request by L0key**
The scoring summary toaster was popping on every request made when the score did not previously exist and then scoring became available for the given domain.  Also adjusted logic to recheck no score available scenarios up to once per day.  Known domain scores will stay on a 30-day timer for scoring recheck and pop.

### CODE REVIEW

File: js/background.js [hunk 1]
```diff
@@ -133,54 +131,51 @@
...
+			message: 'HiddenTabScore',
+			score: data.score,
+			previousScore: data.previousScore,
+			domain: domain
+		});
 	}
 
-	function hasExpired(then, now) {
+	function hasExpired(then) {
 		const expireTime = Number(then) + (expiration.lastScoreExpirationDays * 86400 * 1000);
```
Comment (arlogilbert) [inline js/background.js]: @L0key this looks like it has the same calculation has the hasNoScoreExpired... Supposed to be 30 days?

File: js/background.js [hunk 2]
```diff
@@ -133,54 +131,51 @@
...
+		chrome.tabs.sendMessage( tabId, {
+			message: 'HiddenTabScore',
+			score: data.score,
+			previousScore: data.previousScore,
+			domain: domain
+		});
 	}
 
-	function hasExpired(then, now) {
+	function hasExpired(then) {
```
Comment (arlogilbert) [inline js/background.js:171]: @L0key this looks like it has the same calculation has the hasNoScoreExpired... Supposed to be 30 days?

Comment (L0key) [reply js/background.js:171]: Yep same calc, they just use different constants:
```
const expiration = {
		lastScoreExpirationDays: 30,
		noScoreExpirationDays: 1
	};
```

I'll refactor this a bit...

**Review by arlogilbert — APPROVED** [review]
Whoops, my mistake @L0key

**Pull request merged**